Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -251,5 +284,26 @@ void OnDestroy()
             InteractionManager.SourceUpdated -= InteractionManager_SourceUpdated;
             InteractionManager.SourceLost -= InteractionManager_SourceLost;
         }
+
+        public void OnSelect()
```

Comment:
how do these get called?

Should we make a class that runs in the editor and looks for scripts with void OnSelect() and warn people that way?